### PR TITLE
[Docs] Rest API styles and navigation improvement

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: Lee Calcote <lee.calcote@layer5.io>

**Notes for Reviewers**

This PR improves the readability and navigation of the REST APIs.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 16f6bc2f47b28541c8e8053e06a65fe75adcb9f4 at: https://65c30059eed0262d8753e54f--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
@senali-d @vishalvivekm @sandramsc, thoughts?

File: docs/_sass/swagger.scss [hunk 1]
```diff
@@ -76,17 +82,23 @@ h5 {
 
 .swagger-method-name {
   display: flex;
-  width: 100px;
+  width: 50;
```
Comment (sandramsc) [inline docs/_sass/swagger.scss:85]: This could benefit from a unit of measurement i.e 50**px** or 3.125**rem**?

**Review by senali-d — COMMENTED** [review]
Screenshot 2024-02-08 at 7 23 05 PM

Suggestion: Can we remove the extra space in left side of the description.

**Comment by senali-d** [comment]
The new design looks good.

File: docs/_sass/swagger.scss (hunk 1 above)
Comment (leecalcote) [reply docs/_sass/swagger.scss:85]: Sounds real good, yes.

**Comment by leecalcote** [comment]
@senali-d here is an example of why the text is indented under DESCRIPTION. The same indentation has been added to all heading/sections.


Screenshot 2024-02-08 at 10 18 23 PM



From your screenshot, it looks like the parameters section needs to be indented.

**Comment by l5io** [comment]
🚀 Preview for commit 094a332d9b8b025d085d1e1b6a7a83f2d81bc53c at: https://65c5ad81d060d2582dd05c80--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
I added an indent for the rest of the description details.

**Pull request merged**